src/Frame.js

```diff
@@ -0,0 +1,24 @@
+import SearchBar from './components/SearchBar';
+import Card from './components/Card';
+import './Frame.css';
+
+const Frame = ({className, items}) => {
+  return (
+    <div className={className}>
```

Comment:
컴포넌트가 외부에서 값을 주입받아서 사용하면 범용성이 높아지는 건 맞는데요! 여기서와 같이 사실상 다른 스타일을 적용해서 재사용이 될 가능성이 없는 컴포넌트의 경우에는 굳이 props로 외부에서 주입받는 것 보다는 컴포넌트 내부에서 문자열로 작성해주는 게 좋을 것 같아요.

```suggestion
const Frame = ({ items }) => {
  return (
    <div className='container-frame'>
```